asserts/snap_asserts.go

```diff
@@ -171,18 +172,18 @@ func (snaprev *SnapRevision) SnapDigest() string {
 	return snaprev.Header("snap-digest")
 }
 
-// SnapRevision returns the revision of the snap-id assigned to this build.
-func (snaprev *SnapRevision) SnapRevision() uint64 {
-	return snaprev.snapRevision
+// SnapSize returns the size in bytes of the snap submitted to the store.
+func (snaprev *SnapRevision) SnapSize() uint64 {
+	return snaprev.snapSize
 }
 
-// SnapBuild returns the digest of the associated snap-build.
-func (snaprev *SnapRevision) SnapBuild() string {
-	return snaprev.Header("snap-build")
+// SnapRevision returns the revision assigned to this build of snap-id.
+func (snaprev *SnapRevision) SnapRevision() uint64 {
+	return snaprev.snapRevision
 }
 
-// DeveloperID returns the id of the developer that submitted the snap build to
-// the store.
+// DeveloperID returns the id of the developer that submitted this build of
```

Comment:
Please preserve the original language here as well. This is about a snap build, not a snap-id build.